automation/src/test/scala/org/broadinstitute/dsde/test/api/MethodLaunchSpec.scala

```diff
@@ -33,8 +37,11 @@ class MethodLaunchSpec extends TestKit(ActorSystem("MySpec")) with FreeSpecLike
 
         withMethod("MethodLaunchSpec_input_undefined", MethodData.InputRequiredMethod, 1) { methodName =>
           val method = MethodData.InputRequiredMethod.copy(methodName = methodName)
+          val methodConfigName = createMethodConfigName
+
           Rawls.methodConfigs.createMethodConfigInWorkspace(billingProject, workspaceName, method,
-            method.methodNamespace, methodConfigName, 1, Map.empty, Map.empty, method.rootEntityType)
+            method.methodNamespace, methodConfigName, 1, Map.empty, SimpleMethodConfig.outputs, method.rootEntityType)
+
           val exception = intercept[RestException](Rawls.submissions.launchWorkflow(billingProject, workspaceName, method.methodNamespace, methodConfigName, "participant",
           "participant1", "this", false))
           exception.message.parseJson.asJsObject.fields("message").convertTo[String].contains("Missing inputs:") shouldBe true
```

Comment:
do you think using `eventually` when intercept exception message makes sense to have in tests? I hate tests running in alpha env bc it is notoriously slow.

Reply:
Not in the cases where we expect an exception that originates from rawls. We would have an `eventually` if it was something we'd have to wait for, such as a response from Cromwell, where we might need to wait in some kind of queue and then have to poll. But for rawls, we expect the exception to be thrown immediately on that call, not eventually after some amount of polling. If it is not thrown immediately, then something is wrong and we want the test to fail.